mlir/test/Dialect/Vector/vector-sink.mlir

```diff
@@ -423,3 +423,76 @@ func.func @transpose_elementwise_diff_map_scalable(%a : vector<[4]x6x3x2xf32>, %
   %r = arith.addf %at, %bt : vector<6x[4]x2x3xf32>
   return %r : vector<6x[4]x2x3xf32>
 }
+
+// -----
+
```

Comment:
Please add a block comment documenting which pattern is being tested.

Reply:
done